thirdpartynotices.txt

```diff
@@ -9,6 +9,7 @@ are grateful to these developers for their contribution to open source.
 2. MongoDB (https://github.com/mongodb/node-mongodb-native)
 3. copy-paste (https://github.com/xavi-/node-copy-paste)
 4. opn (https://github.com/sindresorhus/opn)
+5. fs-extra (https://github.com/jprichardson/node-fs-extra)
```

Comment:
Ask Matt to register...

Reply:
I've been doing the registrations myself. It's a super easy form

(I usually complete it after i merge the PR)